### ISSUE TITLE
Can Rust breakpoints work when linked to the Flutter app?

### ISSUE THREAD

**Issue by temeddix** [irondash/cargokit#41]
Compilers may embed some debugging information directly into the compiled code or generate separate files (like .pdb for Windows or .dSYM for macOS) that contain debugging symbols and information. These files aid the debugger in correlating the machine code with the original source code.

Breakpoints will work if we run Rust code directly with `cargo run`, but will they work if they are linked to a Flutter app?

![image](https://github.com/irondash/cargokit/assets/66480156/c3318ee7-0acf-4958-b9d2-653283f5de9a)

**Comment by knopp** [comment]
Yes. Debug build will build the rust code with debug symbols. You can attach to running application and debug the rust code.

**Comment by hilpara** [comment]
@knopp I'm trying to get my workflow setup with flutter_rust_bridge and tested debugging. How ever I could not get this to work. I already wrote about it [here](https://github.com/fzyzcjy/flutter_rust_bridge/issues/2055). 
As I wrote I'm using the [gallery](https://github.com/fzyzcjy/flutter_rust_bridge/tree/master/frb_example/gallery) as an example. On one instance of vscode I started flutter debugger and on the other one I attached to this running process. Attach seems to work, but when I hit the breakpoint, the debugger only shows a message in Debug Console : `Stop reason: signal SIGPROF` and I can't debug anything.

My launch.json:
```
{
            "type": "lldb",
            "request": "attach",
            "pid": "${command:pickProcess}",
            "name": "Debug Rust",
  }
```
So what am I doing wrong?
